peercrawler.py

```diff
@@ -114,6 +114,12 @@ def crawl_once(self):
                 print('Query %41s:%5s (score:%4s)' % ('[%s]' % p.ip, p.port, p.score))
             self.get_peers_from_peer(p)
 
+        # if an inactivity threshold is set the peers considered inactive will be removed here
```

Comment:
Ideally this would be done inside the service's main loop.
If get a copy if never is called, and soemone is sending rubbish data, our list of peers will grow forever without limit.